tools/topology/topology2/cavs-es83x6.conf

```diff
@@ -0,0 +1,259 @@
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:platform>
+<searchdir:platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<virtual.conf>
+<host-gateway-playback.conf>
+<host-gateway-capture.conf>
+<io-gateway.conf>
+<io-gateway-capture.conf>
+<host-copier-gain-mixin-playback.conf>
+<mixout-gain-dai-copier-playback.conf>
+<deepbuffer-playback.conf>
+<dai-copier-be.conf>
+<dai-copier-eqiir-module-copier-capture.conf>
+<gain-capture.conf>
+<gain-module-copier.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<dmic.conf>
+<hda.conf>
+<intel/hw_config_cardinal_clk.conf>
+<manifest.conf>
+<route.conf>
+<intel/common_definitions.conf>
+<dai-copier.conf>
+<module-copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+<dmic-default.conf>
+<hdmi-default.conf>
+<hdmi-in-default.conf>
+IncludeByKey.HDMI_IN_CAPTURE {
+	"true"	"platform/intel/hdmi-in-capture.conf"
+}
+<input_pin_binding.conf>
+<output_pin_binding.conf>
+<input_audio_format.conf>
+<output_audio_format.conf>
+
+Define {
+	MCLK 				24576000
+	# override DMIC default definitions
+	NUM_DMICS			0
+	DMIC0_ID			1
+	DMIC1_ID			2
+	DMIC0_HOST_PIPELINE_ID		13
+	DMIC0_DAI_PIPELINE_ID		14
+	DMIC0_HOST_PIPELINE_SINK	'gain.13.1'
+	DMIC0_DAI_PIPELINE_SRC		'module-copier.14.2'
+	DMIC0_DAI_COPIER		'dai-copier.DMIC.dmic01.capture'
+	DMIC0_DAI_GAIN			'eqiir.14.1'
+	DMIC0_NAME			'dmic01'
+	DMIC0_PCM_CAPS			'Gain Capture 13'
+	# override BE link ids
+	NUM_HDMIS			3
+	HDMI1_ID			3
+	HDMI2_ID			4
+	HDMI3_ID			5
+	PLATFORM 			"none"
+	USE_CHAIN_DMA			"false"
+	DEEP_BUFFER_PIPELINE_ID		15
+	DEEP_BUFFER_PCM_ID		31
+	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
+	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
+	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	HEADSET_CODEC_NAME		'SSP1-Codec'
+	HEADSET_PLAYBACK_PCM_STREAM_NAME 'Headset Playback'
+	HEADSET_CAPTURE_PCM_STREAM_NAME	'Headset Capture'
+	HEADSET_SSP_DAI_INDEX		1
+	HEADSET_HW_CONFIG_NAME		'HEADSET HWCFG'
+	HEADSET_PCM_NAME		"Headset"
+	HEADSET_PCM_ID			0
+}
+
+# override defaults with platform-specific config
+IncludeByKey.PLATFORM {
+	"mtl"	"platform/intel/mtl.conf"
+}
+
+# include DMIC config if needed.
+IncludeByKey.NUM_DMICS {
+	"[1-4]"	"platform/intel/dmic-generic.conf"
+}
+
+# include HDMI config if needed.
+IncludeByKey.NUM_HDMIS {
+	"[3-4]"	"platform/intel/hdmi-generic.conf"
+}
+
+# include deep buffer config if buffer size is in 1 - 1000 ms.
+IncludeByKey.DEEPBUFFER_FW_DMA_MS{
+	"([1-9]|[1-9][0-9]|[1-9][0-9][0-9]|1000)" "platform/intel/deep-buffer.conf"
+}
```

Comment:
why do you need this?

Reply:
Really we are not using in our platform but try to make as similar topology(RT5682) and in future people can use it for other platforms. please let me know if you feel its unnecessary and want to remove it.

Reply:
Please check my above comments and do suggest. Thanks.

Reply:
well, so if the intent is to track an existing topology then adding a new one is really not the direction. There's no such thing as a reference and 'not drift too far', as soon as you fork a new topology it will bit-rot and diverge. 

You'd want to use the same setup really and add the HDMI input that is probably the only thing missing?

Reply:
@plbossart : As you suggested, i have removed the unused feature and uploaded the latest. please review and help to get it merge.